packages/manager/apps/pci-kubernetes/src/components/Loadable.component.tsx

```diff
@@ -0,0 +1,36 @@
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+import { OsdsSpinner } from '@ovhcloud/ods-components/react';
+
+export type TLoadableProps = {
+  when: boolean;
+  spinner?: {
+    centered?: boolean;
+    size?: ODS_SPINNER_SIZE;
+    inline?: boolean;
+  };
+  children?: JSX.Element | JSX.Element[];
+};
+export default function LoadableComponent(
```

Comment:
```suggestion
export default function Loading(
```

i'm not really fond of "LoadableComponent" because i think "Component" should be omitted from the name as it's implicit

what do you think about "Loading"  as a component name? i'm open to other suggestions aswell